pywinauto/unittests/test_mouse.py

```diff
@@ -128,6 +129,37 @@ def test_wheel_click(self):
         self.assertTrue("Mouse Release" in data)
         self.assertTrue("MiddleButton" in data)
 
+    # TODO: make the feature and the tests cross-platform (duration param)
+    if sys.platform == "win32":
+        def test_mouse_can_move_cursor1(self):
+            coord = (20, 10)
+            mouse.move(coord)
+            self.assertEqual(coord, win32api.GetCursorPos())
+
+        def test_mouse_can_move_cursor2(self):
+            coord = (0, 1)
+            mouse.move(coord)
+            self.assertEqual(coord, win32api.GetCursorPos())
+
+        def test_mouse_can_move_cursor3(self):
+            coord = (150, 150)
```

Comment:
Would go on some edge cases instead of repeating with another pair of regular coordinates:
```python
coord = (sys.maxsize, sys.maxsize)
coord = (-200, -300)
coord = (150.0, 150)  # what happens if a float is passed as a coordinate?
```

Reply:
If the coordinate is float, then an error is raised:
TypeError: integer argument expected, got float

Add test?

Reply:
Yep, I would add a test for it

Reply:
Added test_mouse_fail_on_int_duration_and_float_coord 90d87bd172c4a94c7acad7bb9f1a7fd8d0119996